lib/endpoints/class-wp-rest-comments-controller.php

```diff
@@ -699,6 +707,14 @@ protected function prepare_item_for_database( $request ) {
 			$prepared_comment['comment_author_url'] = $request['author_url'];
 		}
 
+		if ( isset( $request['author_ip'] ) ) {
+			if ( filter_var( $request['author_ip'], FILTER_VALIDATE_IP, FILTER_FLAG_IPV4 ) ) {
```

Comment:
What if we added `ip_address` as a new format in our schema, so validation could happen much earlier (and in batch)?